src/notaryUtils.js

```diff
@@ -26,11 +26,15 @@ function createCaseID () {
  */
 function setupCase (params) {
   return co(function * () {
-    const uniqueID = params.caseID || uuid()
+    // the caseID param will be in the format of notary/cases/uuid,
+    // only check if its uuid is less than 40 characters
+    const uniqueID = params.caseID ? params.caseID.substring(params.caseID.indexOf('/cases/') + 7) : uuid()
```

Comment:
What if you use a notary that uses a differently formatted case ID? In REST you should not make any assumptions about what URIs look like internally.

It's true that f-b-sender already makes assumptions about the shape of the URI, but that's precisely why allowing you pass in a custom URI is a good idea. But if you let the user pass in a URI, you should treat it as an opaque string, not try to parse/validate it.

As an example, here is a perfectly valid case URI:

https://bob-notary.com/ilp-notary/123